examples/build.gradle.kts

```diff
@@ -0,0 +1,125 @@
+import com.google.protobuf.gradle.generateProtoTasks
+import com.google.protobuf.gradle.id
+import com.google.protobuf.gradle.ofSourceSet
+import com.google.protobuf.gradle.plugins
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+
+val grpcVersion = "1.30.0"
+val grpcKotlinVersion = "0.1.3"
+val protobufVersion = "3.12.2"
+val coroutinesVersion = "1.3.7"
+
+plugins {
+    application
+    kotlin("jvm") version "1.3.72"
+    id("com.google.protobuf") version "0.8.12"
+    id("org.jlleitschuh.gradle.ktlint") version "9.2.1"
```

Comment:
The addition of this is causing the `ktlintMainSourceSetCheck` to run in the github action, and it's failing because it looks like we weren't previously running it (unfortunately)